test/unit/controllers/CdsErrorHandlerSpec.scala

```diff
@@ -40,22 +42,22 @@ class CdsErrorHandlerSpec extends ControllerSpec with ScalaFutures {
   "Cds error handler" should {
     "redirect to correct page after receive 500 error" in {
       whenReady(cdsErrorHandler.onServerError(mockRequest, new Exception())) { result =>
-        val page = CdsPage(contentAsString(result))
+        val page = CdsPage(contentAsString(Future.successful(result)))
 
-        status(result) shouldBe INTERNAL_SERVER_ERROR
+        result.header.status shouldBe INTERNAL_SERVER_ERROR
```

Comment:
instead of structuring it like this, can we not just pass the future into a val and then pass that into contentAsString and status(...)? Like:

```
val result = cdsErrorHandler.onServerError(mockRequest, new Exception())
val page = CdsPage(contentAsString(result))
status(result) shouldBe INTERNAL_SERVER_ERROR
```